server/docker-base/Dockerfile

```diff
@@ -243,9 +264,27 @@ ENV ANDROID_NDK20_STL_LIB    ${ANDROID_NDK20_PATH}/sources/cxx-stl/llvm-libc++/l
 ENV ANDROID_64_NDK20_STL_LIB ${ANDROID_NDK20_PATH}/sources/cxx-stl/llvm-libc++/libs/arm64-v8a
 ENV ANDROID_NDK20_FILENAME   android-ndk-r${ANDROID_NDK20_VERSION}-linux-x86_64.tar.gz
 
-ENV AAPT ${ANDROID_SDK_BUILD_TOOLS_PATH}/aapt
+ENV PATH ${PATH}:${ANDROID_NDK20_BIN_PATH}
+
+#
+# New NDK25b:
```

Comment:
It's usually good to put the version of Defold where it's effectively in (so in this case 1.4.2)
That way it's easier for us to remove old stuff after 6 months.

Reply:
I'm going to merge it into the next version (after 1.4.2) 
so, I'll add it before merge when I know the next version number